connexion/api.py

```diff
@@ -161,11 +161,15 @@ def add_paths(self, paths=None):
                     if self.debug:
                         logger.exception(error_msg)
                     else:
-                        import sys
                         logger.error(error_msg)
+                        import sys
                         et, ei, tb = sys.exc_info()
-                        raise ei.with_traceback(tb)
-
+                        if sys.version_info[0] == 3:
```

Comment:
Please use six for that as in https://github.com/zalando/connexion/blob/33f7457d1a4851da77f24bc20948a55190543038/connexion/decorators/parameter.py#L29